packages/powersync_attachments_stream/pubspec.yaml

```diff
@@ -0,0 +1,32 @@
+name: powersync_attachments_stream
+description: "A helper library for handling attachments when using PowerSync."
+version: 0.0.1
+repository: https://github.com/powersync-ja/powersync.dart
+homepage: https://www.powersync.com/
+
+environment:
+  sdk: ^3.8.1
+  flutter: ">=1.17.0"
```

Comment:
I think this package shouldn't have to depend on Flutter, right? For those packages you can remove flutter dependencies and the `platforms` section since it will work on everything.